### ISSUE TITLE
Xjadeo not syncing to JACK (Linux)

### ISSUE THREAD

**Issue by NathanIngelb** [x42/xjadeo#56]
Hello,

When using Musescore, it syncs the transport to JACK and I can stop and start with JACK. 
However the video is not playing or starting in Xjadeo. 
It works on my mac but not on Linux Mint (21).

Is there any solution for this?

Many thanks!

Nathan

**Comment by x42** [comment]
Hard to say without additional information. xJadeo should sync to JACK by default...

Can you start it from a Terminal window with the '--verbose' option? That might shed some light on.

**Comment by NathanIngelb** [comment]
xjadeo 0.8.10
OSD font file: /usr/share/xjadeo/ArdourMono.ttf
Video output: OpenGL
GLX-Version : 1.4
enabled drag-DROP support.
GLX: DRI enabled
GLX: use SGI Vblank
selected MIDI driver: JACK-MIDI
using JACK-transport as sync source.
JACK client name: `xjadeo'

entering video update loop @10.00 fps.
Set Fontsize to 170 f:0
smpte:  00:00:00:00 f:0
Bye!

**Comment by x42** [comment]
>  using JACK-transport as sync source.

So this should work. can you check with e.g. qjackctl that JACK transport moves?

**Comment by NathanIngelb** [comment]
Yes, I think it's rolling. Thanks for helping. I added some screenshots of my settings:
![Schermafdruk_2024-04-02_20-52-43](https://github.com/x42/xjadeo/assets/165580787/ab257f0c-6d3f-42d2-9ea8-8a099e73b0a9)
![Schermafdruk_2024-04-02_20-53-02](https://github.com/x42/xjadeo/assets/165580787/e65c42d9-6351-4383-9fe9-f236899939b3)
![Schermafdruk_2024-04-02_20-53-18](https://github.com/x42/xjadeo/assets/165580787/e4ca4dda-5fd0-4152-9859-8d96e927df15)
![Schermafdruk_2024-04-02_20-53-32](https://github.com/x42/xjadeo/assets/165580787/b94cc7b9-4e15-48f8-93f0-e5a1fb35ad27)
![Schermafdruk_2024-04-02_20-54-43](https://github.com/x42/xjadeo/assets/165580787/a42c95d9-42a7-49c7-9214-66b10225ced6)

**Comment by x42** [comment]
So can this be closed?

**Comment by NathanIngelb** [comment]
Sorry, my last post seems a little confusing :)

Robin Gareus schreef op 08-04-2024 15:32:

> So can this be closed?
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>


Links:
------
[1] https://github.com/x42/xjadeo/issues/56#issuecomment-2042769909
[2]

**Comment by NathanIngelb** [comment]
Could it have something to do with video-codecs?

**Comment by x42** [comment]
When you load the video, and offset the time, press "+"  does the video show frames at the given offset?
Or perhaps the video itself has an offset?  try to run  `xjadeo --ignore-file-offset`

Can you try if http://robin.linuxaudio.org/tmp/tsmm2_360_25.webm syncs?


xjadeo 8.10 is also more than 3 years old by now, perhaps try 8.13 (which also supports recent ffmpeg).

**Comment by lorenzosu** [comment]
I am getting a similar issue with version 0.8.13 as packaged in Manjaro (which in turn should come from the Arch package (https://archlinux.org/packages/extra/x86_64/xjadeo/))

xjadeo seems to be totally unresponsive to Jack Transport.
I also noticed that starting it with sync to jack transport spacebar start / pause doesn't work (nor does the menu).

Not sure if relevant, but I am using pipewire. Other jack-transport applications seem to work correctly (I mainly sync Ardour and Rosegarden)

MTC works (tested using Ardour and Rosegarden as sources)

LTC works (tested using Ardour)

This is the output I get with the `--verbose` switch running just with the video file_name (which should use Jack Transport)

```
xjadeo 0.8.13
OSD font file: /usr/share/xjadeo/ArdourMono.ttf
frame rate: 25
duration in seconds: 4185.32
total frames: 104633
file start offset: 0 video-frames
image size: 1920x1080 px
display size: 1920x1080 px
Indexing Video...
First 500 frames are all keyframes. Index disabled. Direkt seek mode enabled.
Good Keyframes 104633
Video output: OpenGL
Scan complete err: 0 use-dts: no
scanned 104633 of 104633 frames, key-int: 1 seek-thresh: 2
max keyframe distance: 74.9 kBytes
Seek by PTS
GLX-Version : 1.4
enabled drag-DROP support.
GLX: DRI enabled
GLX: use SGI Vblank
selected MIDI driver: JACK-MIDI
using JACK-transport as sync source.
JACK client name: `xjadeo'

entering video update loop @25.00 fps.
PTS: Best effort.
smpte:  00:00:00:00 f:0         display:0000000
```

**Comment by x42** [comment]
Can you try if xjadeo 0.8.14 fixes the issue (available from https://xjadeo.sourceforge.net/download.html)

**Comment by lorenzosu** [comment]
> Can you try if xjadeo 0.8.14 fixes the issue (available from https://xjadeo.sourceforge.net/download.html)

Seems fixed for me in 0.8.14 (both by building from source and the binary available at the link)

**Comment by NathanIngelb** [comment]
Yes, it works! Thank you very much!
